app/views/reviews/_index.html.erb

```diff
@@ -9,6 +9,8 @@
   </div>
   <% if review.editable_by?(current_user) %>
     <%= link_to "Edit Review", edit_neighborhood_review_path(review.neighborhood, review)  %>
-    <%= link_to "Delete", neighborhood_review_path(review.neighborhood, review), method: :delete %>
+  <% end %>
+  <% if review.editable_by?(current_user) || (!current_user.nil? && current_user.admin?) %>
```

Comment:
you should not have to be redundant here. Everything after the `||` should be a behavior of the `editable_by?` instance method.

IE:

```ruby
def editable_by?(u)
  u.present? && (u == self.user || u.admin?)
end
```